### PR TITLE
Electron devtools installer

### PR DESCRIPTION
**Pull request by RobinVdBroeck**
This actually is my first pull request to any repo ever, so if I did something wrong please tell me!
- [V] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [V] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [V] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

File: electron-devtools-installer/electron-devtools-installer-tests.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import installExtension, {
+    EMBER_INSPECTOR, REACT_DEVELOPER_TOOLS,
+    BACKBONE_DEBUGGER, JQUERY_DEBUGGER,
+    ANGULARJS_BATARANG, VUEJS_DEVTOOLS,
+    REDUX_DEVTOOLS, REACT_PERF,
+} from 'electron-devtools-installer';
+
+
+installExtension(EMBER_INSPECTOR)
```
Comment (andy-ms) [inline electron-devtools-installer/electron-devtools-installer-tests.ts]: You can just do `const x: Promise<string> = installExtension(EMBER_INSPECTOR);`. Values interpolated in a template aren't typechecked as strings.

File: electron-devtools-installer/electron-devtools-installer.d.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "electron-devtools-installer" {
+    interface ExtensionReference {
+        id: string,
+        electron: string,
+    }
+
+    // Default installation function
+    export default function (extensionReference: ExtensionReference, forceDownload?: boolean): Promise<String>;
```
Comment (andy-ms) [inline electron-devtools-installer/electron-devtools-installer.d.ts]: Prefer `string` to `String`.

Comment (andy-ms) [inline electron-devtools-installer/electron-devtools-installer.d.ts]: According to the docs you can do `installExtension('YOUR_ID_HERE')` too.

**Review by ghost — COMMENTED** [review]
See comments

**Comment by GamesMaxed** [comment]
I changed that. Is it correct now?

**Review by ghost — APPROVED** [review]

**Comment by GamesMaxed** [comment]
Just a quick question, how can I get this to show up in @types?

**Comment by andy-ms** [comment]
Hi @GamesMaxed, changes to the `master` branch aren't automatically published because they have to be manually merged into the `types-2.0` branch first. This happens about once a week; hopefully tomorrow.

**Pull request merged**